tendrl/commons/utils/etcd_utils.py

```diff
@@ -0,0 +1,73 @@
+import etcd
+
+'''
+   Read from etcd
+   input params:
+       _int : type  - >  maps.NamedDict
+              value - >  Ns._int
+       key  : type  - >  string
+              value - >  attr to be fetched
+
+   return param:
+       dict - > if read is successful
+       None - > if key is not found
+'''
+
+def read(_int,key):
```

Comment:
Any specific reasons for having the _int as a parameter??? I feel that's not required @r0h4n can  comment better though